qiskit/synthesis/multi_controlled/mcx_synthesis.py

```diff
@@ -54,6 +54,8 @@ def synth_mcx_n_dirty_i15(
     """
 
     num_qubits = 2 * num_ctrl_qubits - 1
+    if num_ctrl_qubits == 1:
+        num_qubits = 2
```

Comment:
Just a suggestion, but this looks like the nicer flow which avoid re-assigning the `num_qubits` variable
```suggestion
    if num_ctrl_qubits == 1:
        num_qubits = 2
    else:
        num_qubits = 2 * num_ctrl_qubits - 1
```